src/main/java/core/basesyntax/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculateArea {
+    double calculateArea();
+}
```

Comment:
This interface name should be nouns  - `AreaCalculator`

https://howtodoinjava.com/java/basics/java-naming-conventions/